### PR TITLE
Adding Andrew Susanto md and img

### PR DESCRIPTION
**Pull request by AndrewJSusanto**
I believe I did the xml and html file updates correctly, but wasn't sure how to fix the formatting when I ran it locally.

### CODE REVIEW

File: content/member/susanto.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
++++
+bio = ""
+date = "2023-10-11"
+id = "susanto"
+interests = ["XAI", "Autonomous Vehicles", "NLP"]
+name = "Andrew Susanto"
+portrait = "/portraits/leilani.jpg"
```
Comment (lgilpin) [inline content/member/susanto.md:7]: I think you need to change this to andrew :P

Comment (AndrewJSusanto) [reply content/member/susanto.md:7]: Hahaha I will do that! Thank you! Sorry I missed it : P

**Review by AndrewJSusanto — COMMENTED** [review]
Changed it!

**Review by lgilpin — APPROVED** [review]
Looks good

**Pull request merged**